helm/reana/values.yaml

```diff
@@ -52,30 +52,30 @@ components:
     docs_url: http://docs.reana.io
     forum_url: https://forum.reana.io
     imagePullPolicy: IfNotPresent
-    image: reanahub/reana-ui:0.7.0-alpha.1
+    image: reanahub/reana-ui:0.7.0-alpha.2
   reana_db:
     enabled: true
   reana_server:
     imagePullPolicy: IfNotPresent
-    image: reanahub/reana-server:0.7.0-alpha.1
+    image: reanahub/reana-server:0.7.0-alpha.3
     environment:
       REANA_MAX_CONCURRENT_BATCH_WORKFLOWS: 30
     uwsgi:
       processes: 6
       threads: 4
   reana_workflow_controller:
     imagePullPolicy: IfNotPresent
-    image: reanahub/reana-workflow-controller:0.7.0-alpha.1
+    image: reanahub/reana-workflow-controller:0.7.0-alpha.3
     environment:
       SHARED_VOLUME_PATH: /var/reana
   reana_workflow_engine_cwl:
     image: reanahub/reana-workflow-engine-cwl:0.7.0-alpha.1
   reana_workflow_engine_yadage:
-    image: reanahub/reana-workflow-engine-yadage:0.7.0-alpha.1
+    image: reanahub/reana-workflow-engine-yadage:0.7.0-alpha.2
```

Comment:
I see [the tag is called 0.7.0a2](https://hub.docker.com/r/reanahub/reana-workflow-engine-yadage/tags) instead of 0.7.0-alpha.2, is this correct? Looks wrong to me.

Reply:
Git tag is following Python versioning conventions (PEP0440), hence `0.7.0a2`. The produced Docker image is following semantic versioning conventions, hence `0.7.0-alpha.2`. The correspondence is handled by `reana-dev`...

Reply:
Actually, what Marco points to is the Docker tag (see [here](https://hub.docker.com/r/reanahub/reana-workflow-engine-yadage/tags)) which is wrongly following PEP440. Did you release "manually" or did you use `reana-dev`? if so we might have a bug around here? https://github.com/reanahub/reana/blob/e0a29b6e4fb3e6565849e5f7280c340df9477834/reana/reana_dev/release.py#L119-L120

I've recently released [RJC](https://hub.docker.com/r/reanahub/reana-job-controller/tags) using `reana-dev` and it correctly got the semver version...

Reply:
Fixed. 

Hmm, it looks like it must have been manual, but I don't seem to have any entrance at all about tagging `0.7.0a2` in my shell history... Moreover alpha.1 was released well before... So I don't recall what happened.  Let's see next time.

Reply:
... ah, I might know how it happened:

```console
$ cd reana-server
$ git checkout maint-0.7
$ reana-dev docker-build -c . -t auto
reana-dev docker-build -c . -t auto
[2020-09-25T16:13:46] reana-server: git describe --tags
[2020-09-25T16:13:46] reana-server: docker build -t reanahub/reana-server:0.7.0a3 .
^C
Aborted!
```

I must have used `-t auto` probably... I'll create an issue.